package.json

```diff
@@ -1,14 +1,42 @@
 {
   "name": "gridicons",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.2",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "grunt --verbose",
+    "prepublish": "npm run build"
+  },
+  "peerDependencies": {
+    "react": ">=14.x"
```

Comment:
Changes in npm no longer install peer dependencies if they're missing. It's warning correctly in this case. 

We'd add react as a dev dependency if we add component tests in the future.